src/tools/rest-frontend/public/pages/main/entries/form.html

```diff
@@ -0,0 +1,266 @@
+                        </div>
+                        <span class="fa form-control-feedback" style="right:48px;"
+                              ng-class="{'fa-times': entryForm.application.$invalid && entryForm.application.$dirty,
+										 'fa-check': !entryForm.application.$invalid && entryForm.application.$dirty}"></span>
+                        <div ng-show="entryForm.application.$invalid && entryForm.application.$dirty">
+                            <span class="help-block mt5 ml10" ng-messages="entryForm.application.$error">
+                                <span ng-message="required">{{ 'APP.ENTRIES.FORM.LABEL.VALIDATION.APPLICATION.REQUIRED' | translate }}</span>
+                                <span ng-message="minlength">{{ 'APP.ENTRIES.FORM.LABEL.VALIDATION.APPLICATION.MINLENGTH' | translate }}</span>
+                                <span ng-message="maxlength">{{ 'APP.ENTRIES.FORM.LABEL.VALIDATION.APPLICATION.MAXLENGTH' | translate }}</span>
+                                <span ng-message="pattern">{{ 'APP.ENTRIES.FORM.LABEL.VALIDATION.APPLICATION.PATTERN' | translate }}</span>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group has-feedback"
+                     ng-class="{'has-error': entryForm.scope.$invalid && !entryForm.scope.$pristine,
+								'has-success': entryForm.scope.$valid && entryForm.scope.$dirty}">
+                    <label for="scope" class="control-label col-md-3">
+                        {{ 'APP.ENTRIES.FORM.LABEL.SCOPE' | translate }}
+                    </label>
+                    <div class="col-md-9">
+                        <div class="input-group">
+                            <input type="text" id="scope" name="scope" class="form-control"
+                                   ng-model="entry.scope"
+                                   ng-minlength="1" ng-maxlength="255" ng-pattern="'^([a-zA-Z0-9\-\.]+)$'" required
+                                   uib-typeahead="scope for scope in typeaheads.scopes" autocomplete="off"
+                                   placeholder="{{ 'APP.ENTRIES.FORM.PLACEHOLDER.SCOPE' | translate }}">
+                            <span class="input-group-addon">
+                                <i class="fa fa-question form-hint"
+                                   title="{{ 'APP.ENTRIES.FORM.ALTTEXT.SCOPE.HINT' | translate }}"></i>
+                            </span>
+                        </div>
+                        <span class="fa form-control-feedback" style="right:48px;"
+                              ng-class="{'fa-times': entryForm.scope.$invalid && entryForm.scope.$dirty,
+										 'fa-check': !entryForm.scope.$invalid && entryForm.scope.$dirty}"></span>
+                        <div ng-show="entryForm.scope.$invalid && entryForm.scope.$dirty">
+                            <span class="help-block mt5 ml10" ng-messages="entryForm.scope.$error">
```

Comment:
The interesting part is that they are actually wrong, must be a left-over from some refactorization or replacement. The real spacing helpers have the bootstrap-styled format: `md-mt-10`
```
sm ... small
md ... medium
lg ... large
xl ... extra-large

m ... margin
p ... padding
t ... top
b ... bottom
r ... right
l ... left

sm-mt-5 ... margin-top: 5px; on small displays (the setting is <= 480px width)
```

Adding it as comment to the source file.